toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -696,6 +629,17 @@ func (g *PkgGraph) AllNodesFrom(rootNode *PkgNode) []*PkgNode {
 
 // AllRunNodes returns a list of all run nodes in the graph
 func (g *PkgGraph) AllRunNodes() []*PkgNode {
+	nodes := make([]*PkgNode, 0, g.Nodes().Len())
+	for _, n := range g.AllNodes() {
+		if n.Type == TypeLocalRun || n.Type == TypeRemoteRun {
+			nodes = append(nodes, n)
+		}
+	}
+	return nodes
+}
+
+// Returns all RunNodes in the LookupTable
```

Comment:
The comments for public functions should begin with the name of the function. And could we make the comment a bit more detailed, since we now distinguish between all and preferred run nodes?

Reply:
Updated